go.mod

```diff
@@ -4,41 +4,41 @@ go 1.12
 
 require (
 	github.com/alecthomas/participle v1.0.0-alpha2
-	github.com/containerd/containerd v1.4.0-beta.2.0.20200728183644-eb6354a11860
+	github.com/containerd/containerd v1.4.1-0.20201117152358-0edc412565dc
 	github.com/creachadair/jrpc2 v0.8.1
 	github.com/docker/buildx v0.3.2-0.20200410204309-f4ac640252b8
-	github.com/docker/cli v0.0.0-20200227165822-2298e6a3fe24
+	github.com/docker/cli v20.10.0-beta1.0.20201029214301-1d20b15adc38+incompatible
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/docker/go-metrics v0.0.1 // indirect
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/lithammer/dedent v1.1.0
 	github.com/logrusorgru/aurora v0.0.0-20191116043053-66b7ad493a23
-	github.com/mattn/go-isatty v0.0.11
+	github.com/mattn/go-isatty v0.0.12
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/moby/buildkit v0.7.1-0.20200806195445-545532ab0e75
+	github.com/moby/buildkit v0.8.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/openllb/doxygen-parser v0.0.0-20201031162929-e0b5cceb2d0c
 	github.com/pkg/errors v0.9.1
 	github.com/sourcegraph/go-lsp v0.0.0-20200117082640-b19bb38222e2
 	github.com/stretchr/testify v1.5.1
-	github.com/tonistiigi/fsutil v0.0.0-20200724193237-c3ed55f3b481
+	github.com/tonistiigi/fsutil v0.0.0-20201103201449-0834f99b7b85
 	github.com/urfave/cli/v2 v2.1.1
 	github.com/xlab/treeprint v1.0.0
-	golang.org/x/crypto v0.0.0-20200221231518-2aa609cf4a9d
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	google.golang.org/grpc v1.28.0
+	golang.org/x/crypto v0.0.0-20201117144127-c1f2f97bffc9
+	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
+	google.golang.org/grpc v1.29.1
 )
 
 replace github.com/hashicorp/go-immutable-radix => github.com/tonistiigi/go-immutable-radix v0.0.0-20170803185627-826af9ccf0fe
 
 replace github.com/jaguilar/vt100 => github.com/tonistiigi/vt100 v0.0.0-20190402012908-ad4c4a574305
 
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.4.0-beta.2.0.20200728183644-eb6354a11860
+replace github.com/containerd/containerd => github.com/containerd/containerd v1.4.1-0.20201117152358-0edc412565dc
```

Comment:
We can remove both the `containerd/containerd` and `docker/docker` replace statements because buildkit no longer uses them.

Reply:
I think it is our buildx usage that requires this:
```
go: github.com/docker/buildx@v0.3.2-0.20200410204309-f4ac640252b8 requires
        github.com/containerd/containerd@v1.4.0-0: reading github.com/containerd/containerd/go.mod at revision v1.4.0-0: unknown revision v1.4.0-0
```

Reply:
Ah okay. In my `new-debugger` branch, I also updated `buildx`. This is fine then.